fdbserver/VersionedBTree.actor.cpp

```diff
@@ -4775,6 +4984,7 @@ class VersionedBTree {
 
 	// Writes entries to 1 or more pages and return a vector of boundary keys with their ArenaPage(s)
 	ACTOR static Future<Standalone<VectorRef<RedwoodRecordRef>>> writePages(VersionedBTree* self,
+	                                                                        PagerEventReasons reason,
```

Comment:
`writePages()` is for building specifically BTree pages at level `height` which is one of the arguments.  So the `reason` for its writes is always `Commit`, no need to pass it in.